rdr_service/etl/bq/queries.py

```diff
@@ -0,0 +1,951 @@
+  AND stcm1.source_vocabulary_id = 'ppi-sex'
+LEFT JOIN
+  `{dataset_id}.concept` vc1
+ON
+  stcm1.target_concept_id = vc1.concept_id
+  AND vc1.standard_concept = 'S'
+  AND vc1.invalid_reason IS NULL
+GROUP BY
+  src_m.participant_id
+HAVING
+  COUNT(DISTINCT src_m.value_ppi_code) = 1
+     """, 'destination': 'src_gender', 'append': False},
+    'src_mapped': {'query': """
+       SELECT
+   src_c.participant_id                AS participant_id,
+   src_c.date_of_survey                AS date_of_survey,
+   src_c.question_ppi_code             AS question_ppi_code,
+   src_c.question_code_id              AS question_code_id,
+   COALESCE(vc1.concept_id, 0)         AS question_source_concept_id,
+   COALESCE(vc2.concept_id, 0)         AS question_concept_id,
+   src_c.value_ppi_code                AS value_ppi_code,
+   src_c.topic_value                   AS topic_value,
+   src_c.value_code_id                 AS value_code_id,
+   COALESCE(vc3.concept_id, 0)         AS value_source_concept_id,
+   CASE
+      WHEN src_c.is_invalid = 1 THEN 2000000010
+      ELSE COALESCE(vc4.concept_id, 0)
+   END                                 AS value_concept_id,
+   src_c.value_number                  AS value_number,
+   src_c.value_boolean                 AS value_boolean,
+   CASE
+      WHEN src_c.value_boolean = 1 THEN 45877994
+      WHEN src_c.value_boolean = 0 THEN 45878245
+      ELSE 0
+   END                                 AS value_boolean_concept_id,
+   src_c.value_date                    AS value_date,
+   src_c.value_string                  AS value_string,
+   src_c.questionnaire_response_id     AS questionnaire_response_id,
+   src_c.unit_id                       AS unit_id,
+   src_c.is_invalid                    as is_invalid,
+   src_c.src_id                        AS src_id
+FROM `{dataset_id}.src_clean` src_c
+JOIN `{dataset_id}.src_participant` src_p
+   ON  src_c.participant_id = src_p.participant_id
+LEFT JOIN `{dataset_id}.concept` vc1
+   ON  src_c.question_ppi_code = vc1.concept_code
+   AND vc1.vocabulary_id = 'PPI'
+LEFT JOIN `{dataset_id}.concept_relationship` vcr1
+   ON  vc1.concept_id = vcr1.concept_id_1
+   AND vcr1.relationship_id = 'Maps to'
+   AND vcr1.invalid_reason IS NULL
+LEFT JOIN `{dataset_id}.concept` vc2
+   ON  vcr1.concept_id_2 = vc2.concept_id
+   AND vc2.standard_concept = 'S'
+   AND vc2.invalid_reason IS NULL
+LEFT JOIN `{dataset_id}.concept` vc3
+   ON  src_c.value_ppi_code = vc3.concept_code
+   AND vc3.vocabulary_id = 'PPI'
+LEFT JOIN `{dataset_id}.concept_relationship` vcr2
+   ON  vc3.concept_id = vcr2.concept_id_1
+   AND vcr2.relationship_id = 'Maps to value'
+   AND vcr2.invalid_reason IS NULL
+LEFT JOIN `{dataset_id}.concept` vc4
+   ON  vcr2.concept_id_2 = vc4.concept_id
+   AND vc4.standard_concept = 'S'
+   AND vc4.invalid_reason IS NULL
+WHERE src_c.filter = 0
+       """, 'destination': 'src_mapped', 'append': False},
+    'src_meas_mapped': {'query': """
```

Comment:
Can you update the formatting a bit to help with going through the file? We're likely going to need to be able to quickly scan through to find a specific query, so having more separation between them might help. Maybe this could be something like 
```
    ...
    'src_meas_mapped': {
        'query': """
            SELECT ...
```

That could help clarify the structure too.